apps-rendering/src/components/standfirst.tsx

```diff
@@ -93,6 +94,25 @@ const liveblogStyles: SerializedStyles = css`
 	}
 `;
 
+const deadblogStyles = (format: ArticleFormat): SerializedStyles => css`
+	${headline.xxxsmall({ fontWeight: 'bold' })};
+
+	a {
+		text-decoration: none;
+	}
+
+	a:link {
+		color: ${getThemeStyles(format.theme).deadblogStandfirstLink};
```

Comment:
Very minor suggestion - you could save this as a variable and reuse it in the three cases here to save repetition.